model/ftn/w3initmd.ftn

```diff
@@ -2053,16 +2053,33 @@
 !/MPI      IF ((FLOUT(1) .OR. FLOUT(7)).and.(.not. LPDLIB .or.       &
 !/MPI          (GTYPE .ne. UNGTYPE).or. .TRUE.)) THEN
 !
-! NRQMAX is the maximum number of fields, it is the sum of the 
-! sizes of scalar fields (Hs) + 2-component vectors (CUR) + 3-comp ... 
-!
-!/MPI          NRQMAX = 1+0+0 + 19+0+0 + 0+0+0 +                     &
-!/MPI                   2+(NOGE(4)-2)*(NOSWLL+1) + 11+3+1 +          &
-!/MPI                   10+7+1 + 5+4+1 + 5+2+0 + 5+NOEXTR
+! NRQMAX is the maximum number of output fields that require MPI communication,
+! aimed to gather field values stored in each processor into one processor in
+! charge of model output; for each of such fields, this routine requires one
+! call to MPI_SEND_INIT and MPI_RECV_INIT storing the communication request
+! handles in the vectors IRQGO and IRQGO2 respectively.
+! NRQMAX is calculated as the sum of all fields described before (Hs)
```

Comment:
@ukmo-juan-castillo  That's much clearer! Nice one.

Can you just add one comment to explain why most of group 1 is not counted.